elements/lisk-dpos/test/unit/forgers_list_selection.spec.ts

```diff
@@ -32,7 +32,8 @@ import * as forgerSelectionLessTHan103Scenario from '../fixtures/dpos_forger_sel
 import * as forgerSelectionMoreThan2StandByScenario from '../fixtures/dpos_forger_selection/dpos_forger_selection_more_than_2_standby.json';
 import { StateStoreMock } from '../utils/state_store_mock';
 
-describe('Forger selection', () => {
+// TODO: Update after updating protocol-specs
+describe.skip('Forger selection', () => {
```

Comment:
Can you create an issue for this to be done with account schema change?

Reply:
I created issues to update all protocol-specs. How it's related to account schema changes?

Reply:
ah nvm, it's not related. I thought the order will change, but it shouldnt